website/static/js/folderPickerNodeConfig.js

```diff
@@ -427,6 +430,24 @@ var FolderPickerViewModel = oop.defclass({
                 self.loading(false);
                 // Set flag to prevent repeated requests
                 self.loadedFolders(true);
+            },
```

Comment:
Does this need to go in the parent class? Would rather it live in oauthAddonNodeConfig